Corzbank.Data/Entities/User.cs

```diff
@@ -1,11 +1,17 @@
-﻿using System;
+﻿using Microsoft.AspNetCore.Identity;
+using System;
 using System.Collections.Generic;
 using System.Text;
 
 namespace Corzbank.Data.Entities
 {
-    public class User: KeyEntity
+    public class User: IdentityUser
     {
-        public int MyProperty { get; set; }
+        public string Firstname { get; set; }
+      
+        public string Lastname { get; set; }
+
+
```

Comment:
We already discussed this several times. Remove odd lines
**One empty line between properties**

Reply:
resolved